### PR TITLE
Featur/infopage

### PR DESCRIPTION
**Pull request by janneserichsen**
Hello Guys in this Pull i redesigned my Folder Structure added the Infopage 
and made my Feed Page dynamical

### CODE REVIEW

File: components/list/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// onClick={() => {
+// setTitle("");
+//onAddTodo(title);
+// }}
+//  >
+//     Add
+//   </button>
+//  </>
+//  );
+//}
```
Comment (niklasrammelkamp) [inline components/list/index.js:22]: Why not delete this page if you comment everything out?

File: components/Comment/index.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+// <button onClick={() => setIsEditing(true)}>Edit</button>
+// </>
+// );
+// }
+// <label>
+//   {todoContent}
+//   <button onClick={() => onDelete(todo.id)}>Delete</button>
+//   </label>
+//);
+//}
```
Comment (niklasrammelkamp) [inline components/Comment/index.js:93]: Why not delete this page if you comment everything out?

File: pages/_app.js [hunk 3]
```diff
@@ -1,14 +1,3 @@
...
-      <GlobalStyle />
-      <Head>
-        <title>Capstone Project</title>
-      </Head>
-      <Component {...pageProps} />
-    </>
-  );
+export default function App({ Component, pageProps: { pageProps } }) {
+  return <Component {...pageProps} />;
 }
```
Comment (niklasrammelkamp) [inline pages/_app.js:3]: Why did you delete the `Head`?

```suggestion
export default function App({ Component, pageProps }) {
  return (
    <>
      <Head>
        <title>Capstone Project</title>
      </Head>
      <Component {...pageProps} />
    </>
  );
}
```

File: pages/buttonexample/index.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
+import React from "react";
```
Comment (niklasrammelkamp) [inline pages/buttonexample/index.js:1]: you can delete this

File: pages/buttonexample/index.js [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+import reactCSS from "reactcss";
+import { SketchPicker } from "react-color";
+
+class ButtonExample extends React.Component {
```
Comment (niklasrammelkamp) [inline pages/buttonexample/index.js:5]: How does this component work? I've never seen this before. 
Why does the button is named `ButtonExample`? If it's the final button you should name it differently.
I don't want to imply anything but it seems like a copy from somewhere 'cause we never had this notation in our sessions. 
I think that's no problem but if you did that you should write a comment about where you got this from.

File: pages/index.js [hunk 6]
```diff
@@ -1,3 +1,172 @@
...
+    comment: "#Long live Alsen",
+  },
+  {
+    id: "g-318Lcidaded",
+    userID: "u-318Lciad",
+    location: "Itzehoe",
+    image: "/images/asie-wall.png",
+    comment: "#Long live Alsen",
+  },
+];
```
Comment (niklasrammelkamp) [inline pages/index.js:145]: You can outsource this in a file like `store.js` and then import it here.

File: pages/index.js [hunk 7]
```diff
@@ -1,3 +1,172 @@
...
+              width={250}
+              height={250}
+            />
+            <p>{graffiti.comment}</p>
+          </section>
+        );
+      })}
+      <section>
+        <Navbar />
+      </section>
```
Comment (niklasrammelkamp) [inline pages/index.js:169]:
```suggestion
        <Navbar />
```

File: pages/profile/index.js [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+          {`
+            .header {
+              font-size: 25px;
+              text-align: center;
+              border: 4px;
+              border-radius: 4px;
+              border-color: grey;
+            }
+          `}
+        </style>
```
Comment (niklasrammelkamp) [inline pages/profile/index.js:17]: You should use StyledComponents instead

File: pages/styles/Discover.css [hunk 9]
```diff
@@ -0,0 +1,5 @@
+.section {
+  border: 3px;
+  border-color: black;
+  border-radius: 4px;
+}
```
Comment (niklasrammelkamp) [inline pages/styles/Discover.css:5]: We wanna use StyledComponents

**Review by niklasrammelkamp — COMMENTED** [review]
Hi Jannes, I reviewed your code and found some issues. Pleas check and let me know if I wrote something wrong

File: components/list/index.js (hunk 1 above)
Comment (janneserichsen) [reply components/list/index.js:22]: because i want to use this comment function later and i had some trouble with it cause i changed some code

File: components/Comment/index.js (hunk 2 above)
Comment (janneserichsen) [reply components/Comment/index.js:93]: because i want to use this comment function later and i had some trouble with it cause i changed some code

File: pages/_app.js (hunk 3 above)
Comment (janneserichsen) [reply pages/_app.js:3]: actually i dont know im gonna fix that

File: pages/buttonexample/index.js (hunk 5 above)
Comment (janneserichsen) [reply pages/buttonexample/index.js:5]: okay gonna do this yes i copied the code

File: pages/profile/index.js (hunk 8 above)
Comment (janneserichsen) [reply pages/profile/index.js:17]: allready fixed this
